tests/integration/linux_bridge_test.py

```diff
@@ -269,6 +304,15 @@ def test_replace_port_on_linux_bridge(port0_vlan101, port1_up):
         )
 
 
+def test_linux_bridge_over_bond_over_slave(bond0, bridge0):
+    bridge_state = bridge0[Interface.KEY][0]
+    desired_state = deepcopy(bond0)
+    desired_state[Interface.KEY].append(bridge_state)
```

Comment:
I can also perform this 'state merge' in an added fixture, which would:
  - combine/merge states
  - apply desired state
  - yield desired state

I could actually move the 'finally' clauses of the bridge0 and bond0 fixtures to the finally clause of that new fixture.

What do you think @EdDev  ?

Reply:
The fixtures represent the test setup and teardown. They are not part of the test.
Therefore, I think it is fine as it is.

Reply:
> The fixtures represent the test setup and teardown. They are not part of the test.
Therefore, I think it is fine as it is.

touchè.